lib/team_api/joiner.rb

```diff
@@ -138,7 +138,7 @@ def join_project_data
     def store_project_errors(project, errors)
       project['errors'] = errors
       name = project['github'][0] || project['name']
-      data['errors'][name] |= errors unless data['errors'].nil?
+      data['errors'][name] = (data['errors'][name] || []) | errors
```

Comment:
Yeah. The first one should work. Originally, I wanted to preserve any errors that were already in the data, but that is unnecessary. Based on where this is in the process, the errors in `data` are the same as `project`.